src/main/frontend/src/components/aggregates/AggregateDialog.svelte

```diff
@@ -127,12 +110,12 @@
 	<ConsumerExchange bind:consumerExchangeName bind:receivers bind:methods />
 	<CardActions>
 		{#if editMode}
-			<Button class="mr-3" on:click={update} disabled={!valid}>Update</Button>
+			<Button class="mr-3" on:click={update} disabled={!valid}>Update</Button> <!--TODO: vlingo colors: class="primary-color", right now materialify overwrites disabled state-->
```

Comment:
I hope I can fix that

Reply:
it's because there are scss variables in `_material-theme.scss` which breaks svelte-materialify. css structure.
Simple solution can be `class={disabled ? 'disabled-class' : 'default-class'}`

Reply:
yeah, but that's a failure of materialify, I'll see if upstream can change it